lesson_06/lesson_06.3/task_02.js

```diff
@@ -0,0 +1,68 @@
+const OPERATION = "+-*/%^";
+const firstNumber = prompt("Please enter first number.");
+const operationSymbol = prompt("Please select operation +, -, *, /, % or ^");
+const secondNumber = prompt("Please enter second number.");
+const isNumber = function checkInput(enterNumber) {
+  if (
+    typeof Number(enterNumber) === "number" &&
+    enterNumber !== null &&
+    !isNaN(enterNumber) &&
```

Comment:
```suggestion
    !Number(isNaN(enterNumber)) &&
```
так буде краще, дивись https://github.com/airbnb/javascript#standard-library--isnan

Reply:
Виправив.